src/main/scala/me/chuwy/otusfp/home_work/Main.scala

```diff
@@ -0,0 +1,39 @@
+package me.chuwy.otusfp.home_work
+
+import cats.data.EitherT
+import cats.effect.{ExitCode, IO, IOApp}
+import cats.implicits._
+import me.chuwy.otusfp.home_work.bootstrap.Environment
+import me.chuwy.otusfp.home_work.config.{loadConfig}
+import me.chuwy.otusfp.home_work.server.HttpServer
+import me.chuwy.otusfp.home_work.model.AppError
+import cats.effect.std.Console
+import me.chuwy.otusfp.home_work.logging.Logger
+
+object Main extends IOApp {
+  type F[A] = EitherT[IO, AppError, A]
+
+  override def run(args: List[String]): IO[ExitCode] = {
+    (for {
+      config <- loadConfig[F]
+      env    <- Environment.initialize[F](config)
+      _      <- env.loggerF.info(s"Запуск сервера с параметрами $config")
+      _      <- start(env) as (ExitCode.Success)
```

Comment:
```suggestion
      _      <- start(env).as(ExitCode.Success)
```

Просто предупреждаю что коммьюнити по-тихоньку избавляется от такого синтаксиса (кажется 2.13.6 даже должен выводить ворнинги)